### PR TITLE
docs: Fix broken link from unpack parser to pack stage

### PR DESCRIPTION
**Pull request by chaudum**
### What this PR does / why we need it

**before**:

**after**:

### CODE REVIEW

**Review by MichelHollands — APPROVED** [review]
LGTM

File: docs/sources/clients/promtail/stages/pack.md [hunk 1]
```diff
@@ -57,7 +57,7 @@ This would create a log line
 }
 ```
 
-**Loki 2.2 also includes a new [`unpack`](../../../../logql/log_queries/#unpack) parser to work with the pack stage.**
+**Loki 2.2 also includes a new [`unpack` parser]({{< relref "../../../logql/log_queries.md#unpack" >}}) to work with the pack stage.**
```
Comment (kavirajk) [inline docs/sources/clients/promtail/stages/pack.md:60]: why `relref` here?

**Review by kavirajk — APPROVED** [review]
LGTM

File: docs/sources/clients/promtail/stages/pack.md (hunk 1 above)
Comment (chaudum) [reply docs/sources/clients/promtail/stages/pack.md:60]: `relref` is the preferred way to link documents.
First, it throws an error if the linked page does not exist, preventing 404s. And second, it is agostic about how the actual URL of the linked document looks.

**Review by vlad-diachenko — APPROVED** [review]
LGTM

**Pull request merged**